### DISCUSSION TITLE
2023-06-06 개발일지

### DISCUSSION THREAD

**Discussion by Craftcode17** [Blue-club/pintos1#12] [개발일지]
(●'◡'●)

**Comment by Riudiu** [comment]
오늘 한 것
- Argument Passing 구현

![스크린샷 2023-06-06 오후 5 38 21](https://github.com/Riudiu/pintos/assets/86466976/6ffbd9e9-6d53-4476-b611-975c06a74239)

 - 시스템 콜 구현 시작
